synapse/tests/test_lib_storm.py

```diff
@@ -678,6 +678,58 @@ def handler(evt):
             # We have to know queue names to add nodes too
             self.raises(BadSyntaxError, core.eval, 'inet:ipv4 task()')
 
+    def test_storm_delprop(self):
+        with s_cortex.openurl('ram:///') as core:
+            t0 = core.formTufoByProp('inet:fqdn', 'vertex.link', created="20170101")
+            self.isin('inet:fqdn:created', t0[1])
+            # Operator syntax requires force=1
+            core.eval('inet:fqdn=vertex.link delprop(:created)')
+            t0 = core.getTufoByProp('inet:fqdn', 'vertex.link')
+            self.isin('inet:fqdn:created', t0[1])
+
+            core.eval('inet:fqdn=vertex.link delprop(:created, force=0)')
+            t0 = core.getTufoByProp('inet:fqdn', 'vertex.link')
+            self.isin('inet:fqdn:created', t0[1])
+
+            core.eval('inet:fqdn=vertex.link delprop(:created, force=1)')
+            t0 = core.getTufoByProp('inet:fqdn', 'vertex.link')
+            self.notin('inet:fqdn:created', t0[1])
+
+            # Re-add the prop and delete the prop via the macro syntax directly
+            t0 = core.setTufoProps(t0, created="20170101")
+            self.isin('inet:fqdn:created', t0[1])
+            core.eval('inet:fqdn=vertex.link [ -:created ]')
+            t0 = core.getTufoByProp('inet:fqdn', 'vertex.link')
+            self.notin('inet:fqdn:created', t0[1])
+
+            # Delete muliple props via macro syntax
+            t0 = core.setTufoProps(t0, created="20170101", updated="20170201")
+            self.isin('inet:fqdn:created', t0[1])
+            self.isin('inet:fqdn:updated', t0[1])
+            core.eval('inet:fqdn=vertex.link [ -:created  -:updated]')
+            t0 = core.getTufoByProp('inet:fqdn', 'vertex.link')
+            self.notin('inet:fqdn:created', t0[1])
+            self.notin('inet:fqdn:updated', t0[1])
+
+            # Cannot delete "ro" props via delprop
+            t1 = core.formTufoByProp('inet:netuser', 'vertex.link/pennywise')
+            self.isin('inet:netuser:user', t1[1])
+            # Operator syntax requires force=1
+            result = core.ask('inet:netuser [ -:user ]')
```

Comment:
I'd add another test here that doesn't use the macro syntax and has force enabled.

ex: `result = core.ask('inet:netuser delprop(:user, force=1)')`

Reply:
this is a good catch!  added